nss-day-cohort-51/nutshell-ghoul-diggers
# Friends Task Articles Sort Complete

# Description

Articles tab only displays either the current users posts or their friends posts with different styling.

## Type of change



# Testing Instructions
To test the new addition add friends to your friends list to see how the posts are filtered now.

# Checklist:
